core/src/main/java/org/apache/calcite/rel/core/TableModify.java

```diff
@@ -65,7 +66,34 @@ public abstract class TableModify extends SingleRel {
    * Enumeration of supported modification operations.
    */
   public enum Operation {
-    INSERT, UPDATE, DELETE, MERGE
+
+    //We assign each enum an integer value. This is used when converting "MERGE INTO" into a
+    // LogicalTableModify node, to enumerate which action should be taken for each row.
```

Comment:
I just googled how to convert an enum to an integer, I don't think it's legal to cast an enum value in a situation where you need an integer.

Reply:
To clarify. I'm more asking/suggesting that we should always use `Operator.INSERT.value` (or whatever the Java way to get the value is) instead of the hardcoded values. I just want to confirm we are doing that and there is no alternative section in any code base (calcite, BodoSQL, or Bodo) that must be kept consistent. If so we need to directly link to the other section of the code in all locations (but BodoSQL should hopefully be able to import these values from Calcite).